pkg/bootstrap/types.go

```diff
@@ -56,3 +56,17 @@ type CloudBootstrapResult struct {
 	PlatformType   configv1.PlatformType
 	PlatformRegion string
 }
+
+type FlowsConfig struct {
+	// Target IP:port of the flow collector
+	Target string
+
+	// CacheActiveTimeout is the max period, in seconds, during which the reporter will aggregate flows before sending
+	CacheActiveTimeout *uint
```

Comment:
why not just have them as uint or int? Size of uint or int is the same as size of *uint on 64-bit systems. This looks a very C-like syntax.

Reply:
The main reason was that all the properties here are optional and 0 are meaningful values which might not coincide with the defaults.

Reply:
Ok makes sense.